defaults/main.yml

```
@@ -4,6 +4,7 @@ node_exporter_binary_local_dir: ""
 node_exporter_web_listen_address: "0.0.0.0:9100"
 
 node_exporter_textfile_dir: "/var/lib/node_exporter"
+node_exporter_telemetry_path: "/metrics"
```

```suggestion
node_exporter_web_telemetry_path: "/metrics"
```

Could you also move it 2 lines up so it would be closer to `node_exporter_web_listen_address`?